docs/pages/api-docs/step-button.md

```diff
@@ -30,6 +30,8 @@ The `MuiStepButton` name can be used for providing [default props](/customizatio
 |:-----|:-----|:--------|:------------|
 | <span class="prop-name">children</span> | <span class="prop-type">node</span> |  | Can be a `StepLabel` or a node to place inside `StepLabel` as children. |
 | <span class="prop-name">classes</span> | <span class="prop-type">object</span> |  | Override or extend the styles applied to the component. See [CSS API](#css) below for more details. |
+| <span class="prop-name">completed</span> | <span class="prop-type">bool</span> |  | For non-linear Steppers you need to manually set which steps are completed. Otherwise the Stepper determines if a step is completed. |
+| <span class="prop-name">disabled</span> | <span class="prop-type">bool</span> |  | If `true`, the base button will be disabled. |
```

Comment:
.This shouldn't appear. Looking into it.

Reply:
Removing this would obscure the types too much. It's unfortunate but I don't have time to spend hours on this. Leaving a special note in IntelliSense may help.